### PR TITLE
Refactor the ContainerPanel class with MVC in mind

### PR DESCRIPTION
**Pull request by madoar**
This PR refactors the `ContainerPanel` class with MVC in mind. For this the `ContainerPanel` class has been replaced with a `ContainerInformationPanel` component, which needs to be wrapped in a `DetailsPanel` object.

### CODE REVIEW

**Comment by madoar** [comment]
In addition to the comments in #1865:

It is interesting to note that the bindings don't seem to be invalidated (triggered) by:
https://github.com/PhoenicisOrg/phoenicis/blob/41de4372050185e467eeb634be12ff571917d9be/phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java#L134-L137

I'm not sure why this is the case. Therefore to "update" the tabs in the container details panel, the container needs to be reselected in the list widget.

**Comment by plata** [comment]
> Therefore to "update" the tabs in the container details panel, the container needs to be reselected in the list widget.

Is this a new issue introduced in this PR?

**Comment by madoar** [comment]
No this hasn't been introduced by this PR. The source for this behavior is the way we pass new information to the `ContainerInformationPanel`, which only occurs when selecting an entry in the list widget.

**Comment by madoar** [comment]
The newest commit fixes #1865.

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java [hunk 1]
```diff
@@ -51,18 +47,12 @@
...
 
     private boolean firstViewSelection = true;
 
-    public ContainersController(ContainersView containersView,
-            ContainersManager containersManager,
-            ContainerEngineController containerEngineController,
-            RepositoryManager repositoryManager,
-            EngineSettingsManager engineSettingsManager,
-            VerbsManager verbsManager,
+    public ContainersController(ContainersView containersView, ContainersManager containersManager,
```
Comment (plata) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java]: I think this was more readable before.

Comment (madoar) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/containers/ContainersController.java]: Ok, I've reverted the constructor parameter style for the class.

**Pull request merged**